help/learn/Custom-Machine-Learning.md

```diff
@@ -0,0 +1,132 @@
+# Custom ML models
```

Comment:
Hi @Denomet , thanks for contributing to our wiki, this is always welcome! If you don't mind, I'll leave some general comments on how we format our articles.
So, first, let's change the file name to lowercase words separated by dashes, e.g., `custom-ml-models.md` (it's typically the same as the article name). Also, we use HTML comments at the top of the text to annotate it, so the first line here should be:

```
<!-- TITLE: Custom ML Models -->
```

This way we determine how the article title should look like in the table of contents.